script.js

```diff
@@ -0,0 +1,369 @@
+      state5 =false;
+      return function () {return counter += 1; }
+    })();
+
+    add();
+
+    }
+    else if(state5 ==true){
+
+
+      var add = (function () {
+        document.getElementById("cell5").style.color = "Blue";
+        document.getElementById("cell5").innerHTML = "X";
+state5 =false;
+        return function () {return counter += 1;}
+      })();
+
+      add();
+
+}
+
+else{ }
+
+
+}
+
+function myFunction6() {
+
+
+    if(counter % 2==0 && state6 ==true)
+    {
+
+    var add = (function () {
+      document.getElementById("cell6").style.color = "Blue";
+      document.getElementById("cell6").innerHTML = "0";
+      state6 =false;
+      return function () {return counter += 1; }
+    })();
+
+    add();
+
+    }
+    else if(state6 ==true){
+
+
+      var add = (function () {
+        document.getElementById("cell6").style.color = "Blue";
+        document.getElementById("cell6").innerHTML = "X";
+state6 =false;
+        return function () {return counter += 1;}
+      })();
+
+      add();
+
+}
+
+else{ }
+
+
+}
+
+function myFunction7() {
+
+
+    if(counter % 2==0 && state7 ==true)
+    {
+
+    var add = (function () {
+      document.getElementById("cell7").style.color = "Blue";
+      document.getElementById("cell7").innerHTML = "0";
+      state7 =false;
+      return function () {return counter += 1; }
+    })();
+
+    add();
+
+    }
+    else if(state7 ==true){
+
+
+      var add = (function () {
+        document.getElementById("cell7").style.color = "Blue";
+        document.getElementById("cell7").innerHTML = "X";
+state7 =false;
+        return function () {return counter += 1;}
+      })();
+
+      add();
+
+}
+
+else{ }
+
+
+}
+
+function myFunction8() {
+
+
+    if(counter % 2==0 && state8 ==true)
+    {
+
+    var add = (function () {
+      document.getElementById("cell8").style.color = "Blue";
+      document.getElementById("cell8").innerHTML = "0";
+      state8 =false;
+      return function () {return counter += 1; }
+    })();
+
+    add();
+
+    }
+    else if(state8 ==true){
+
+
+      var add = (function () {
+        document.getElementById("cell8").style.color = "Blue";
+        document.getElementById("cell8").innerHTML = "X";
+state8 =false;
+        return function () {return counter += 1;}
+      })();
+
+      add();
+
+}
+
+else{ }
+
+
+}
+
+function myFunction9() {
+
+
+    if(counter % 2==0 && state9 ==true)
+    {
+
+    var add = (function () {
+      document.getElementById("cell9").style.color = "Blue";
+      document.getElementById("cell9").innerHTML = "0";
+      state9 =false;
+      return function () {return counter += 1; }
+    })();
+
+    add();
+
+    }
+    else if(state9 ==true){
+
+
+      var add = (function () {
+        document.getElementById("cell9").style.color = "Blue";
+        document.getElementById("cell9").innerHTML = "X";
+state9 =false;
+        return function () {return counter += 1;}
+      })();
+
+      add();
+
+}
+
+else{ }
+
+
+}
```

Comment:
I like that you were able to make this work - but try to resist the urge to copy and paste this many times. Try starting with just two cells and DRY'ing up your code then before proceeding.